Source/WebCore/page/LocalDOMWindow.cpp

```diff
@@ -2715,15 +2715,24 @@ void LocalDOMWindow::dispatchPendingEventTimingEntries()
 {
     auto renderingTime = performance().nowInReducedResolutionSeconds();
     if (m_pendingPointerDown && !m_pendingPointerDown->duration)
-        m_pendingPointerDown->duration = renderingTime - m_pendingPointerDown->startTime;
+        m_pendingPointerDown->duration = std::max(renderingTime - m_pendingPointerDown->startTime, Seconds::fromMilliseconds(1));
+
+    for (auto& keydownEntry : m_pendingKeyDowns) {
+        if (!keydownEntry.value.keyDown.duration)
```

Comment:
Should we be using `std::optional<>` instead of relying on 0 as a magic number for durations?